__mocks__/handlers/task-details.handler.ts

```diff
@@ -2,6 +2,48 @@ import { rest } from 'msw';
 const URL = process.env.NEXT_PUBLIC_BASE_URL;
 
 const taskDetailsHandler = [
+    rest.get(`${URL}/tasks/12345/details`, (_, res, ctx) => {
+        try {
+            return res(
+                ctx.status(200),
+                ctx.json({
+                    message: 'task returned successfully',
+                    taskData: {
+                        id: '12345',
+                        isNoteworthy: true,
+                        lossRate: {
+                            dinero: 0,
+                            neelam: 0,
+                        },
+                        purpose: 'This is a sample description',
+                        endsOn: 1618790410,
+                        title: 'test 1 for drag and drop',
+                        status: 'COMPLETED',
+                        assignee: 'ankur',
+                        links: [],
+                        dependsOn: [],
+                        percentCompleted: 0,
+                        type: 'feature',
+                        priority: 'high',
+                        featureUrl: 'https://www.sampleUrl.com',
+                        startedOn: 1617062400,
+                        completionAward: {
+                            neelam: 0,
+                            dinero: 110,
+                        },
+                        github: {
+                            issue: {
+                                html_url:
+                                    'https://github.com/sample-org/sample-repo/issues/000',
+                            },
+                        },
+                    },
+                })
```

Comment:
Hmm.., Looks like we can extract this big json in some variable

Reply:
accepted.